4.x/amd/dojo/package.json

```diff
@@ -23,12 +23,12 @@
     "gulp": "^4.0.2",
     "gulp-clean": "^0.4.0",
     "gulp-rename": "^1.2.2",
-    "node-sass": "~4.14.1",
+    "sass": "^1.30.0",
     "uglify-js": "^3.6.0"
```

Comment:
Can this be removed?

Reply:
We still have a sass build guide, but I can get Franco to review that to make sure it's up to date

Reply:
Sorry, I meant can uglify-js be removed?